### ISSUE TITLE
Who's the real killer? - Request for comments

### ISSUE THREAD

**Issue by Bost** [CSSEGISandData/COVID-19#634]
I want to release an enhancement of my chatbot https://t.me/corona_cases_bot showing an estimated number of road kills "side by side" with corona cases. What do you think about that? If it understandable / wise / etc.?

-----------
**12 Mar 2020** @corona_cases_bot  Worldwide      /zz    /zzz

**188630** Road kills since 22 Jan (estimated) based on [2018 WHO estimation](https://www.who.int/publications-detail/global-status-report-on-road-safety-2018) (calculated for 2016); see [Death on the roads](https://extranet.who.int/roadsafety/death-on-the-roads/)

**Corona cases**:
Confirmed: 128343
Sick: 55299  ~ 43%  
Recovered: 68324  ~ 53%  
Deaths: 4720  ~ 4%        See [mortality rate](https://www.worldometers.info/coronavirus/coronavirus-death-rate/) and /references
Closed: 73044  ~ 57%  = recovered + deaths

-----------

![image](https://user-images.githubusercontent.com/1174677/76630526-798fc900-6540-11ea-9a74-0709bf3899ca.png)

**Comment by Codeplukkertje** [comment]
What are you trying to achieve with this?

**Comment by shirriff** [comment]
Comments:
a) This is not an issue with the CSSE data, so the issue doesn't belong here and should be closed.
b) Traffic deaths are important and people should pay more attention to them (in my opinion), but it seems entirely irrelevant to your chatbot. But it's your chatbot so do what you want.
c) For a North American audience, "road kill" generally refers to dead animals, not human deaths so it's going to confuse people.

**Comment by intrigus-lgtm** [comment]
IMHO there is no point in comparing traffic accidents and corona.
Traffic accidents don't spread exponentially.

**Comment by Bost** [comment]
> Traffic accidents don't spread exponentiaĽudia

Neither does the death rate of this disease. Nevertheless some people are panicking as is we all gonna die. The purpose of my enhancement is to help understand that.

**Comment by Bost** [comment]
> issue doesn't belong here

@shirriff  Can you point me to the right discussion forum? Thanks in advance.

**Comment by Bost** [comment]
> What are you trying to achieve with this?

To show you that (except for Italy at the moment) buckling your seatbelt, not speeding etc. is even more important than regularly washing hands. And that this claim is backed by numbers.

**Comment by iandoug** [comment]
Do you actually understand exponential growth?

**Comment by Bost** [comment]
> > What are you trying to achieve with this?
> 
> To show you that (except for Italy at the moment) buckling your seatbelt, not speeding etc. is even more important than regularly washing hands. And that this claim is backed by numbers.

In Italy since the "beginning" in January 22 there have  been more Corona deaths than road fatalities.

**Comment by Bost** [comment]
> Do you actually understand exponential growth?

And do you understand how many people never heard about "exponential growth" ?
Really. Give me your best guess - a number just to see how good is your understanding of the world around you.

Like, have you tried to ask, let say your grandmother about exponential growth? The world is more than just developers and engineers.

**Comment by Shane073** [comment]
@Bost my best guess for a conversation like this, you would be best off going to twitter or reddit. Car crashes have nothing to do with COVID-19 data.

**Comment by iandoug** [comment]
> And do you understand how many people never heard about "exponential growth" ?

> Like, have you tried to ask, let say your grandmother about exponential growth? The world is more than just developers and engineers.

"The greatest shortcoming of the human race is our inability to understand the exponential function." 
 -- Albert Allen Bartlett

As pointed out in the beginning above, you're trying to compare a linear line (car deaths) with an exponential curve. In the beginning the linear line will win. But not in the end.
But judging by the votes on comments here, it seems people agree with me that you don't understand exponential growth. 
There are many things (eg donkeys) that kill more people per year than Covid-19 has, so far.
But for example, the UK Gov has figured that they're okay with half a million dead, just in their country. It won't happen overnight... the 1918 flu took 18 months to do its damage.

**Comment by Shane073** [comment]
@iandoug I saw Alberts youtube video on exponential growth years ago. "The Most Important Video You'll Ever See". The title wasn't wrong. https://www.youtube.com/watch?v=DZCm2QQZVYk

**Comment by Bost** [comment]
> you're trying to compare a linear line (car deaths) with an exponential curve.

No. Not at all. 
First of all, since there is not an endless number of people on Earth the function can't be exponential. It can be at most logistic, as very well explained here https://youtu.be/Kas0tIxDvrg?t=392
But that's not the point.

> In the beginning the linear line will win. But not in the end.

So for example in China:

![image](https://user-images.githubusercontent.com/1174677/76662615-c85c5380-657e-11ea-8d35-9c279f5b4046.png)

No sign of any exponential curve near of far. At most logistic - for confirmed cases. And pretty soon also for the recovered cases. As everybody hopes.

Whereas the REAL problem - the DEAD not the ill people - all I see is a linear progression for corona deaths - the red line, and some other much steeper (albeit still linear) progression. And no one likes to talk about this one. The real elephant in the room!

> you're trying to compare a linear line (car deaths) with an exponential curve.

So again, I'm comparing two linear progressions here. Can you see it now?

> UK Gov has figured that they're okay with half a million dead, just in their country. It won't happen overnight... the 1918 flu took 18 months to do its damage.

Oh, that's interesting, send a link please.

**Comment by Shane073** [comment]
This is the start of an exponential curve. It forms a "hockey stick". Exclude China's data and see what you get. 
![Cases and deaths outside china](https://user-images.githubusercontent.com/36460101/76664296-f6ba4d00-65d7-11ea-8c0d-c095b46dea14.png)

Mate, if you want to compare car crash fatalities to COVID-19, go for it. As others said, this isn't the forum for it.

**Comment by Bost** [comment]
And maybe a better example - imagine a situation where we have two parallel events going on:
-  an asteroid size of an Egyptian pyramid on collision course with Earth, having the potential to wipe out **half** of the Earths population
- a coronavirus outbreak - like we have it - with steep, some may call it exponential progression in its early days, killing in the end let's say the same number of people as the Spanish flu did. 50,000,000. (Or some similar, in the context of the size of population our planet, rather negligible number.)

The fatalities for first event will look like straight, almost horizontal "curve", better described as CONSTANT line whereas the curvature of the second of the events is steep indeed.

Now, should we focus our attention according to the shapes of graphs or rather according to absolute number of fatalities? That's my point.

**Comment by iandoug** [comment]
@Bost 
Okay, there are bigger issues at play here.
Firstly, there is no political agenda in car crash figures.
For covid, there are various factors screwing up the data, so comparing it against anything else is not apples-vs-apples.
For example, North Korea has no cases. Really? Or Turkey recently added 1 or 2. Really?

China is worse. They changed the way they register cases at least twice, hence the strange bump in the middle of their curve. Then politicians decided the country looked bad/incompetent and unable to cope, so they fired a few officials "for not bringing the numbers down". THE VERY NEXT DAY, daily increase numbers started decreasing. Go figure how accurate they were.

Worse, once the hospitals were full in Hubei, people were left to die at home, those numbers never got added into the official figures. So what should have been exponential growth suddenly wasn't.

The UK has decided to stop bulk testing and only test those in hospitals. Sweden the same. So we no longer get accurate case counts from these Model Western Nations.

So these things are going to mess up any curves we draw, and also any hope at accurate CFR numbers. Politics over People.

The UK numbers I mentioned were discussed here:
https://www.youtube.com/watch?v=BYTFk34nhoI

Basically their sum is: 66million people, 80% infected, 1% of those die. Comes to around half a million.

Cheers, Ian
